tests/test_rotation.py

```diff
@@ -122,3 +123,16 @@ def testAngleDist(self):
         self.assertTrue(
             np.allclose(angles, angular_dist, atol=utest_tolerance(self.dtype))
         )
+
+    def testFromRotvec(self):
```

Comment:
Since the method was converted to scipy. We might want to have a less tautological test also.

I see this but looks like a lot of work:
https://en.wikipedia.org/wiki/Rotation_matrix#Rotation_matrix_from_axis_and_angle

How about pick an x,y, or z axis, rotate by some random angle and check the rotation matrix directly? (or with about_axis?) That should be quick I think.

Reply:
done.